packages/fee-estimation/README.md

```diff
@@ -0,0 +1,112 @@
+# GasEstimation Documentation
+
+## Why
+
+Gas estimation on OP-chains has both an l2 and l1 component. By default tools such as Viem, Wagmi, Ethers, and Web3.js do not support the l1 component. They will support this soon but in meantime this library can help estimate fees for transactions.
+As these tools add support for gas estimation natively this README will be updated with framework specific instructions.
+
+## High level
+
+- The l2 contract that can estimate l1Fees is called [GasPriceOracle](../contracts-bedrock/contracts/l2/GasPriceOracle.sol) contract. This library provides utils for interacting with it at a high level.
+- The GasPriceOracle is [deployed to Optimism](https://optimistic.etherscan.io/address/0x420000000000000000000000000000000000000F) and other OP chains at a predeployed address of `0x420000000000000000000000000000000000000F`
+
+For more detailed information about gas fees on Optimism's Layer 2, you can visit their [official documentation](https://community.optimism.io/docs/developers/build/transaction-fees/#the-l2-execution-fee).
```

Comment:
Should mention that the main method here is estimateFees since 9/10 folks using this library only care about that one function

Reply:
agreed! calling at the function first and foremost